src/js/components/NewArticle.js

```diff
@@ -20,4 +20,4 @@ export default class NewArticle extends Component {
       <ArticleForm handleSubmit={ this.handleSubmit.bind(this) } />
     )
   }
-}
\ No newline at end of file
+}
```

Comment:
Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(